org-formation/600-access/_tasks.yaml

```diff
@@ -387,3 +387,18 @@ SynapseLlmDeveloperPolicy:
         ]
       }
     PolicyName: SynapseLlmDeveloperPolicy
+
+# Setup cross-account access to Image Builder Pipeline in Image Central account from Synapse accounts
+SynapseImageCentralImageBuilderPipelineAccess:
```

Comment:
This name seems overly specific to me.  I would remove `Synapse` because I think it would be useful to allow all member accounts access to this role.